receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -47,19 +49,24 @@ type transaction struct {
 	ctx           context.Context
 	isNew         bool
 	sink          consumer.MetricsConsumer
+	jobsMap       *map[string]*map[string]*metricspb.TimeSeries
```

Comment:
may be another type for map[string]*metricspb.Timeseries would make it more readable.
```go

type timeseriesMap map[string]*metricspb.Timeseries
type jobsCache map[string]*timeseriesMap

type transaction struct {
    jobsMap *jobsCache
}
```

Reply:
Done